modules/app.js

```diff
@@ -11,6 +12,7 @@ const addBtn = document.querySelector('.add');
 
 storage = JSON.parse(localStorage.getItem('books')) || [];
 
+//  ADD BOOK HERE
 function addBook(book, title, author) {
   book.title = title;
   book.author = author;
```

Comment:
- [ ] Lastly, the functionality to add the book in your project seems not to be working. This may be due to the refactoring you have done. Please correct it 🙏